### ISSUE TITLE
Timeline not updating

### ISSUE THREAD

**Issue by patricksaber** [Duna/TV-EPG-Android-Recyclerview#16]
Timeline not updating when scrolling or pressing now button if the time is 9:00 and the time changed to 9:05 the time line stay at 9:00

**Comment by Duna** [comment]
Yes it is true, the touch reacts on the sides of now indicator, can you fix it?
